uk_bin_collection/uk_bin_collection/councils/ForestOfDeanDistrictCouncil.py

```
@@ -1,132 +1,142 @@
+                            or td.get_text(strip=True)
+                        )
+                        if container_type and raw_date:
+                            data["bins"].append({
+                                "type": container_type,
+                                "collectionDate": self._parse_date(raw_date, current_year),
+                            })
+                    except (ValueError, AttributeError):
+                        continue
+            else:
+                body_text = driver.find_element(By.TAG_NAME, "body").text
+                lines = [l.strip() for l in body_text.split("\n") if l.strip()]
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)